extensions.json

```
@@ -117,8 +117,8 @@
     {
       "id": "arcanis.vscode-zipfs",
       "repository": "https://github.com/yarnpkg/berry",
-      "version": "2.3.1-rc.3",
-      "checkout": "vscode-zipfs/2.3.1-rc.3",
+      "version": "2.4.0-rc.2",
+      "checkout": "@yarnpkg/parsers/2.4.0-rc.2",
```

Wrong auto-detected tag:

```suggestion
      "version": "2.4.0-rc.2",
      "checkout": "vscode-zipfs/2.4.0-rc.2",
```